packages/invoices/src/InboundShipment/DetailView/columns.ts

```diff
@@ -9,12 +9,13 @@ import {
   useColumns,
   Column,
 } from '@openmsupply-client/common';
-import { InvoiceLine, InboundShipmentItem } from './../../types';
+import { InboundItem } from './../../types';
```

Comment:
Consolidating on using the `item` nested object rather than the flattened fields. It's easier to just take the fields from the nested object rather than have fields on one and others nested..